----
try {
throw 2;} catch(x) { alert (x); }
----
try
  throw 2
catch x
  alert x
